plugins/modules/system/vdo.py

```diff
@@ -465,7 +477,8 @@ def run_module():
         biothreads=dict(type='str'),
         cputhreads=dict(type='str'),
         logicalthreads=dict(type='str'),
-        physicalthreads=dict(type='str')
+        physicalthreads=dict(type='str'),
+        force=dict(type='bool', default=False)
```

Comment:
```suggestion
        force=dict(type='bool', default=False),
```
To improve the diff the next time something is added :)